doc/ss_logger.rst

```diff
@@ -0,0 +1,201 @@
+    11:15:00 system.host.com SmartSim[130033] INFO This is a message
+    11:15:00 system.host.com SmartSim[130033] ERROR This is an error message
+    11:15:00 system.host.com SmartSim[130033] WARNING This is a warning message
+
+Notice that the `debug` function message was filtered. This is because by using
+a higher severity level (`info`), we instruct SmartSim to omit the lower severity levels (`debug` and `developer`).
+
+Next, set `SMARTSIM_LOG_LEVEL` to `debug`::
+    export SMARTSIM_LOG_LEVEL=debug
+
+When we execute the script,
+the following messages will print to stdout::
+    11:15:00 system.host.com SmartSim[65385] INFO This is a message
+    11:15:00 system.host.com SmartSim[65385] DEBUG This is a debug message
+    11:15:00 system.host.com SmartSim[65385] ERROR This is an error message
+    11:15:00 system.host.com SmartSim[65385] WARNING This is a warning message
+
+Notice that all log messages print to stdout. By using a lower severity level (`debug`),
+we instruct SmartSim to print all log functions at and above the level.
+
+Next, set `SMARTSIM_LOG_LEVEL` to `developer`::
+    export SMARTSIM_LOG_LEVEL=developer
+
+When we execute the script,
+the following messages will print to stdout::
+    11:15:00 system.host.com SmartSim[65385] INFO This is a message
+    11:15:00 system.host.com SmartSim[65385] DEBUG This is a debug message
+    11:15:00 system.host.com SmartSim[65385] ERROR This is an error message
+    11:15:00 system.host.com SmartSim[65385] WARNING This is a warning message
+
+Notice that all log messages print to stdout. By using the lowest severity level (`developer`),
+we instruct SmartSim to print all log messages. Remember that `developer` is extremely verbose
+logging and is intended for use during development.
+
+Next, set `SMARTSIM_LOG_LEVEL` to `quiet` in terminal::
+    export SMARTSIM_LOG_LEVEL=quiet
+
+When we run the program once again, the following output is printed
+to stdout::
+    11:15:00 system.host.com SmartSim[65385] ERROR This is an error message
+    11:15:00 system.host.com SmartSim[65385] WARNING This is a warning message
+
+Notice that the `info` and `debug` log functions were filtered. This is because by using
+the highest severity level (`quiet`), we instruct SmartSim to omit the lower severity levels
+(`info`, `debug` and `developer`).
+
+To finish the example, set `SMARTSIM_LOG_LEVEL` to `info` in terminal::
+    export SMARTSIM_LOG_LEVEL=info
+
+When we execute the script, the following messages will print
+to stdout::
+    11:15:00 system.host.com SmartSim[130033] INFO This is a message
+    11:15:00 system.host.com SmartSim[130033] ERROR This is an error message
+    11:15:00 system.host.com SmartSim[130033] WARNING This is a warning message
+
+Notice that the same messages were logged to stdout as when we ran the script with the default value `info`.
+SmartSim omits the lower severity levels (`debug` and `developer`).
```

Comment:
SmartSim omits the lower severity levels (`debug` and `developer`).

->

SmartSim omits the high verbosity level messages (`debug` and `developer`).